build/webpack/webpack.finsemble-built-in.entries.json

```diff
@@ -33,7 +33,7 @@
 	},
 	"yesNoDialog": {
 		"output": "components/yesNoDialog/yesNoDialog",
-		"entry": "./src-built-in/components/yesNoDialog/src/app.jsx"
+		"entry": "./src-built-in/components/yesNoDialog/yesNoDialog.jsx"
```

Comment:
The other components have a file called "app.jsx". Why change it?

Reply:
Just to be more clear - to void having a bunch of app.jsx files to sort through when you search for a file

Reply:
I totally agree with you, but we should be consistent throughout the whole app as a matter of polish. Maybe we can make a quick card to fix all of these.

Reply:
Yup. I created a card to address that.